dash_table/FormatTemplate.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+from .Format import Format, Sign, Symbol, Type
+
+def money(decimals: int = 2, sign: Sign = Sign.UNDEFINED):
+    decimals = decimals if isinstance(decimals, int) else 2
+
+    return Format() \
+    .precision(decimals) \
+    .sign(sign) \
+    .symbol(Symbol.CURRENCY) \
+    .type(Type.FIXED_POINT) \
+    .create()
```

Comment:
`\` is generally to be avoided - I suppose if we add constructor kwargs that'll do it here, but using this same syntax you can just wrap the whole expression in parentheses and the `\` will be unneeded.